SwiftWisdom/Core/StandardLibrary/Array/Array+Utilities.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 public extension Array {
-    public func ip_subArrayFromIndices(_ indices: [Int]) -> [Element] {
+    public func ip_subArrayFrom(indices: [Int]) -> [Element] {
```

Comment:
I'm not sure if this is needed since `indices` is visible by default now. So it will read `ip_subArrayFrom(indices: [Int])`